src/serialupdi.c

```diff
@@ -165,6 +165,11 @@ static int serialupdi_decode_sib(const PROGRAMMER *pgm, updi_sib_info *sib_info)
       updi_set_nvm_mode(pgm, UPDI_NVM_MODE_V3);
       updi_set_datalink_mode(pgm, UPDI_LINK_MODE_24BIT);
       break;
+    case '5':
+      pmsg_notice("NVM type 5: 24-bit, page oriented (alias type 3)\n");
```

Comment:
I would post this as a warning, explaining that even though chip is reporting NVM v5, we are still going to use NVM v3 anyway.

Reply:
I agree.

Reply:
The wording will be corrected